sandbox/src/mocks/PasRespondedWithErrorOperationOutcome.json

```diff
@@ -0,0 +1,24 @@
+{
+  "meta": {
+    "profile": [
+      "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-OperationOutcome-1"
+    ]
+  },
+  "resourceType": "OperationOutcome",
+  "issue": [
+    {
+      "severity": "error",
+      "code": "exception",
+      "details": {
+        "coding": [
+          {
+            "system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-APIErrorCode-1",
+            "code": "PAS_RESPONDED_WITH_ERROR",
+            "display": "Error recieved from PAS"
+          }
+        ]
+      },
+      "diagnostics": "PAS unable to create booking into slot '11002026' for UBRN: 000000070000. PAS Request Status: ERROR"
```

Comment:
It's probably not great that we're leaking the PAS request status out from our APIs, might be worth creating a tech debt around removing this from the diagnostic message.